src/ocamlorg_web/lib/redirection.ml

```diff
@@ -194,9 +194,9 @@ let from_v2 =
     ( "/learn/tutorials/comparison_of_standard_containers.zh.html",
       Url.tutorial "data-structures-comparison" );
     ( "/learn/tutorials/compiling_ocaml_projects.ja.html",
-      Url.tutorial "compiling-ocaml-projects" );
+      Url.tutorial "using-the-ocaml-compiler-toolchain" );
```

Comment:
```suggestion
      Url.tutorial "using-the-ocaml-compiler-toolchain" );
    (Url.tutorial "compiling-ocaml-projects",
      Url.tutorial "using-the-ocaml-compiler-toolchain");
```

Since we're renaming the slug, we also need to introduce a new redirect from the old page to the new one.

Reply:
Hey, CI check is failing. Could this be because there is an irregularity in the pattern `(url_string, Url.tutorial "tutorial_name")`?